proposals/lip-0027.md

```diff
@@ -903,6 +903,14 @@ There are no incompatibilities since the protocol is not changed. Applications o
 }
 ```
 
+## Reference Implementation
+
+1. [Pablo Vicente](https://github.com/pablitovicente): https://github.com/LiskHQ/lisk-sdk/pull/5322
+2. [Shusetsu Toda](https://github.com/shuse2): https://github.com/LiskHQ/lisk-sdk/issues/5214
+3. [Ishan Tiwari](https://github.com/ishantiw): https://github.com/LiskHQ/lisk-sdk/pull/5385
+4. [Nazar Hussain](https://github.com/nazarhussain): https://github.com/LiskHQ/lisk-sdk/pull/5396
```

Comment:
Strictly speaking I think https://github.com/LiskHQ/lisk-sdk/pull/5385 and https://github.com/LiskHQ/lisk-sdk/pull/5396 is not related to the codec implementation
Also, https://github.com/LiskHQ/lisk-sdk/issues/5214 is issue.
maybe list below PRs as those are main PRs
https://github.com/LiskHQ/lisk-sdk/pull/5309
https://github.com/LiskHQ/lisk-sdk/pull/5319
https://github.com/LiskHQ/lisk-sdk/pull/5322
https://github.com/LiskHQ/lisk-sdk/pull/5326
https://github.com/LiskHQ/lisk-sdk/pull/5339
https://github.com/LiskHQ/lisk-sdk/pull/5334
https://github.com/LiskHQ/lisk-sdk/pull/5341
https://github.com/LiskHQ/lisk-sdk/pull/5350